testdata/p4_16_samples_outputs/parser-unroll/parser-unroll-uninitialized-slice-read.p4-stderr

```diff
@@ -0,0 +1,12 @@
+parser-unroll-uninitialized-slice-read.p4(5): [--Wwarn=uninitialized_use] warning: f may be uninitialized
+        g = f[3:0];
+            ^
+parser-unroll-uninitialized-slice-read.p4(3): [--Wwarn=uninitialized_out_param] warning: out parameter 'f' may be uninitialized when 'p' terminates
+parser p(packet_in packet, out bit<8> f, out bit<4> g) {
+                                      ^
+parser-unroll-uninitialized-slice-read.p4(3)
+parser p(packet_in packet, out bit<8> f, out bit<4> g) {
+       ^
+parser-unroll-uninitialized-slice-read.p4(5): [--Wwarn=ignore-prop] warning: Result of 'g = f[3:0]' is not defined: Error: Uninitialized
+        g = f[3:0];
+          ^
```

Comment:
The fact that this warning is worded this way is not great - I opened https://github.com/p4lang/p4c/issues/4945 to improve this and similar warnings